nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/syslog/SyslogReader.java

```diff
@@ -57,18 +57,31 @@
         "Note: Be mindfull that RFC3164 is informational and a wide range of different implementations are present in" +
         " the wild.")
 public class SyslogReader extends SchemaRegistryService implements RecordReaderFactory {
+
     public static final String GENERIC_SYSLOG_SCHEMA_NAME = "default-syslog-schema";
     static final AllowableValue GENERIC_SYSLOG_SCHEMA = new AllowableValue(GENERIC_SYSLOG_SCHEMA_NAME, "Use Generic Syslog Schema",
             "The schema will be the default Syslog schema.");
+
+    static final String RAW_MESSAGE_NAME = "_raw";
+
     public static final PropertyDescriptor CHARSET = new PropertyDescriptor.Builder()
             .name("Character Set")
             .description("Specifies which character set of the Syslog messages")
             .required(true)
             .defaultValue("UTF-8")
             .addValidator(StandardValidators.CHARACTER_SET_VALIDATOR)
             .build();
+    public static final PropertyDescriptor ADD_RAW = new PropertyDescriptor.Builder()
```

Comment:
Same here, the property isn't added to the list of supported property descriptors